src/applications/disability-benefits/686c-674/config/helpers.js

```diff
@@ -58,8 +52,8 @@ export const ServerErrorAlert = (
     </p>
     <p className="vads-u-font-size--base">
       If you get this error again, please call the VA.gov help desk at{' '}
-      <Telephone contact={CONTACTS.VA_311} /> (TTY:{' '}
-      <Telephone contact={CONTACTS['711']} pattern={PATTERNS['3_DIGIT']} />
+      <va-telephone contact={CONTACTS.VA_311} /> (TTY:{' '}
+      <va-telephone contact={CONTACTS['711']} />
```

Comment:
The `va-telephone` component has a [`tty` property](https://design.va.gov/storybook/?path=/docs/components-va-telephone--default#tty) that can be used instead of adding `TTY:`

Reply:
Updated with the last commit. It slightly alters the markup on the TTY component. 
![Screenshot 2023-07-12 at 10 30 10 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/1820743/6fa53a50-e5d2-4acd-81d6-5b2e89eece46)